src/components/bike/BikeBookmark.vue

```diff
@@ -0,0 +1,77 @@
+<script lang="ts">
+import { Icon, IconType } from '@/components/icon'
+import { defineComponent, type PropType } from 'vue'
+
+export default defineComponent({
+  name: 'BikeBookmark',
+  components: {
+    Icon
+  },
+  props: {
+    active: {
+      type: Boolean,
+      default: false
+    },
+    outlined: {
+      type: Boolean,
+      default: false
+    },
+    size: {
+      type: String as PropType<UISizing | UISizingType>,
+      default: 'base'
+    },
+    width: {
+      type: Number,
+      default: 0
+    }
+  },
+  emits: ['update:active'],
```

Comment:
If possible, please define parameters for emits, for better type safety.
```suggestion
  emits: {
    'update:active': (__value: boolean) => true
  },
```